platform/src/main/resources/db/migration/V6__sep24_field_updates.sql

```diff
@@ -2,7 +2,7 @@ ALTER TABLE sep24_transaction ADD sep10_account_memo VARCHAR(255);
 
 ALTER TABLE sep24_transaction ADD amount_expected VARCHAR(255);
 
-ALTER TABLE sep24_transaction ADD kyc_verified BOOLEAN SET default FALSE;
+ALTER TABLE sep24_transaction ADD kyc_verified BOOLEAN;
```

Comment:
I think we need `ADD COLUMN`, not just `ADD`, right?

Reply:
Actually it looks like this may be optional.

Reply:
`ADD COLUMN` is optional.